compile.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+COMPILE_TARGET_INPUT_FILE=$1
+
+echo "compiling ${COMPILE_TARGET_INPUT_FILE} ..."
+
+COMPILE_TARGET_OUTPUT_FILE=${COMPILE_TARGET_INPUT_FILE//.cpp/}
+COMPILE_TARGET_OUTPUT_FILE=${COMPILE_TARGET_OUTPUT_FILE##*\/}
+
+clang++ -O2 -std=c++11 -Werror -o bin/${COMPILE_TARGET_OUTPUT_FILE} ${COMPILE_TARGET_INPUT_FILE}
```

Comment:
Why stop at c++11 when all modern compilers support c++17?

Reply:
I did not remember why I typed C++11, but it supposed to be C++17 :-)